File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car{

    Map <String, Map<String , Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    

    public UberBlack (String license, Account driver, Map<String, 
                      Map<String,Integer>> typeCarAccepted,ArrayList<String> seatsMaterial ){

        super(license, driver);// With this, I call to the costructor method from the parent_class
        this.seatsMaterial=seatsMaterial;
       // this.typeCarAccepted =typeCarAccepted;

         }
}

File: PHP/uberBlack.php
<?php
require_once('car.php');
class UberBlack extends Car { //this line shows the inheritance, Uber black inherits the Class parent class car
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial){ //onstructor from the child class
        parent::__construct($license,$driver); // constructor to be met as the parent class had constructor
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }


}
?>